pkgs/tools/typesetting/tex/texlive/combine.nix

```diff
@@ -35,11 +37,8 @@ let
 
   name = "texlive-${extraName}-${bin.texliveYear}${extraVersion}";
 
-  texmfroot = (buildEnv {
-    name = "${name}-texmfroot";
-
-    # the 'non-relocated' packages must live in $TEXMFROOT/texmf-dist (e.g. fmtutil, updmap look for perl modules there)
-    extraPrefix = "/texmf-dist";
+  texmfdist = (buildEnv {
+    name = "${name}-texmfdist";
```

Comment:
just to make sure: Is the reversal of #226536 intentional?

Reply:
Yes, it's a partial reversal: we still build a TEXMFROOT, but we need separate `buildEnv`'s for TEXMFDIST and the `tlpkg` directory. So `texmfdist` goes back to what it was before and `texmfroot` becomes the other derivation below.